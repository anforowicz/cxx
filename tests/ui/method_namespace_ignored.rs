#[cxx::bridge]
mod ffi {
    unsafe extern "C++" {
        type SomeReceiver;

        #[namespace = "this_will_be_ignored_and_therefore_should_be_an_error"]
        fn some_method(self: &SomeReceiver) -> bool;
    }
}

fn main() {}
