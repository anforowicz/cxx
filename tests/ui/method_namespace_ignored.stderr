error: Ignored `namespace` attribute - will use receiver's namespace instead
 --> tests/ui/method_namespace_ignored.rs:6:9
  |
6 |         #[namespace = "this_will_be_ignored_and_therefore_should_be_an_error"]
  |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
